flask_appbuilder/templates/appbuilder/general/lib.html

```diff
@@ -75,7 +75,11 @@
 		{% for action_key in actions %}
         {% set action = actions.get(action_key) %}
 				$('.{{action.name}}_menu_item').on('click', function(){
-				    return modelActions.execute_multiple('{{action.name}}','{{action.confirmation}}');
+                    {% if confirmation %}
```

Comment:
`confirmation` comes from where?

Reply:
I don't know, I supposed suggestion from @mik-laj was correct.
Let me do some check.

Reply:
It should be action.confirmation